prov/verbs/src/fi_verbs.c

```diff
@@ -897,8 +897,13 @@ static int fi_ibv_getinfo(uint32_t version, const char *node, const char *servic
 	int ret;
 
 	ret = fi_ibv_init_info(hints);
-	if (ret)
-		return ret;
+	if (ret) {
+		/* If there were no devices, that's not an error */
+		if (ret == -ENODEV)
+			return -FI_ENODATA;
+		else
+			return ret;
+	}
```

Comment:
I pushed a change to my PR to handle this.  Can you drop this portion of the patch?